assets/app/views/browse/route.html

```diff
@@ -0,0 +1,74 @@
+<div ng-controller="RouteController" class="content">
+  <project-page>
+    <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+    <alerts alerts="alerts"></alerts>
+    <div ng-if="route">
+      <div class="col-md-12">
+        <div class="tile">
+          <dl class="dl-horizontal left">
+            <dt>Name:</dt>
+            <dd>{{route.metadata.name}}</dd>
+            <dt>Labels:</dt>
+            <dd>
+              <span ng-if="!route.metadata.labels">none</span>
```

Comment:
It is but it is set to null when empty so the key isn't even in the api
response
On Oct 2, 2015 4:40 PM, "Sam Padgett" <notifications@github.com> wrote:

> In assets/app/views/browse/route.html
> <https://github.com/openshift/origin/pull/4921#discussion_r41065272>:
>
> > @@ -0,0 +1,74 @@
> > +<div ng-controller="RouteController" class="content">
> > +  <project-page>
> > +    <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
> > +    <alerts alerts="alerts"></alerts>
> > +    <div ng-if="route">
> > +      <div class="col-md-12">
> > +        <div class="tile">
> > +          <dl class="dl-horizontal left">
> > +            <dt>Name:</dt>
> > +            <dd>{{route.metadata.name}}</dd>
> > +            <dt>Labels:</dt>
> > +            <dd>
> > +              <span ng-if="!route.metadata.labels">none</span>
>
> Is route.metadata.labels an object? (route.metadata.labels | hashSize)
> === 0?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4921/files#r41065272>.
>